test/calico/metadata_access_test.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build packet aws aws_edge
+// +build e2e
+
+package calico_test
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/yaml"
+
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+)
+
+const (
+	// Define manifest as YAML and then unmarshal it to Go stuct, so it is easier to
+	// write and debug, as it can be copy-pasted to the YAML file and applied manually.
```

Comment:
>as it can be copy-pasted to the YAML file and applied manually

To *which* YAML file? Is this a well-known file? Assuming it isn't, it should be "to **a** YAML file". The intention isn't the same in both cases.

Reply:
Yes, let me fix :)